### PR TITLE
[ENG-162] Updated readme file 

### PR DESCRIPTION
**Pull request by shivani-nicatch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,78 @@
-# seroai-trusthub
\ No newline at end of file
+# Seroai Trusthub
```
Comment (trustCoder101) [inline README.md]: Sero AI

File: README.md [hunk 2]
```diff
@@ -1 +1,78 @@
...
+<p align="center">
+   <a href="https://github.com/seroai-coders/seroai-trusthub/stargazers"><img src="https://img.shields.io/github/stars/seroai-coders/seroai-trusthub" alt="Github Stars"></a>
+   <a href="https://github.com/seroai-coders/seroai-trusthub/issues"><img src="https://img.shields.io/github/issues/seroai-coders/seroai-trusthub" alt="Github issues"></a>
+    <a href="https://github.com/seroai-coders/seroai-trusthub/blob/main/LICENSE"><img src="https://img.shields.io/github/license/seroai-coders/seroai-trusthub" alt="license"></a>
+</p>
+
+## 🛠️ Built With
+
+- [Next.js](https://nextjs.org)
+- [Tailwind CSS](https://tailwindcss.com)
```
Comment (trustCoder101) [inline README.md]: should we add daisyui and tremor as well?

File: README.md [hunk 3]
```diff
@@ -1 +1,78 @@
...
+- [Prisma](https://www.prisma.io)
+- [TypeScript](https://www.typescriptlang.org)
+
+## ✨ Getting Started
+
+Please follow these simple steps to get a local copy up and running.
+
+### Prerequisites
+
+- Node.js (Version: >=18.x)
```
Comment (trustCoder101) [inline README.md]: I think we have 20.x . Please confirm. Thanks!

**Review by trustCoder101 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (shivani-nicatch) [reply README.md]: @trustCoder101 Changed it to v21.x.

File: README.md (hunk 2 above)
Comment (shivani-nicatch) [reply README.md]: Added both

**Pull request merged**